src/main/resources/data/minecolonies/tags/items/mechanic_ingredient.json

```diff
@@ -7,6 +7,7 @@
     "minecraft:blaze_rod",
     "minecraft:ender_pearl",
     "minecraft:ender_eye",
-    "minecraft:redstone_torch"
+    "minecraft:redstone_torch",
+    "minecraft:glowstone_dust"
```

Comment:
If we're allowing compression to blocks here, do we also want to allow decompression, and add the glowstone block?

Reply:
It's possible, but that almost feels more of a Crusher job, especially since the Crusher's better equipped to give better output rates as the building is upgraded.

Reply:
Yeah, I agree that the crusher might be the best place. If I remember correctly, vanilla doesn't have a block -> dust recipe, but  many modpacks do. If we enable the recipe at the crusher, it will be the first 'taught' recipe that the crusher has, unless we want to enable this as a custom recipe that allows it in vanilla. That last I'd like to hold off on, as that starts to go down the path of non-deterministic inputs to output ratios, which probably won't be supported for a while.